doc/widgets/imageembedding.rst

```diff
@@ -34,7 +34,7 @@ vector for each image. It returns an enhanced data table with additional columns
 Images can be imported with :doc:`Import Images <importimages>` widget or as paths to images in a spreadsheet. In this case the column with images paths needs a three-row header with *type=image* label in the third row.
 
 .. figure:: images/header-example.png
-   :scale: 50%
+   :width: 100%
```

Comment:
This should exactly scale the image, since they are all taken at full res and appear exceedingly large when docs are built. Hence scaling, not setting width.....

Reply:
Perhaps I am not understanding sth...

Reply:
I see the problem now, but width doesn't solve the issue of excessively large images in the docs... Which scale does, but then it ruins resizing... :( What to do?

Reply:
The problem appears when a window is narrow (see the screenshot below). I will try to find the solution that combines booth.  

![screenshot 2017-12-29 11 52 11](https://user-images.githubusercontent.com/6421558/34435560-ec676636-ec8e-11e7-9082-f1f82ec95005.png)

Reply:
Yeah, I can see it now. I just hate having widget images so big (for examples, that's not a real problem). We might as well just scale images manually and call it a day....

Reply:
I just found out that the scale argument actually does not change the resolution of images while building the documentation. Scale influence the width and height argument in HTML but images are still loaded in their original size. So better solution is to scale the images manually.

Reply:
This was intended. Scale is not meant to resize the actual image, but just to make it clearer to read the docs. I like it when I can click on an image and it loads in full res (especially important for example images). We already index images to keep down the size...

Reply:
I understand your point now. Full width (100 %) would make images too big when looking at the documentation in a wide window. Scale sets the fixed image sizes in HTML what causes squeezing problem. I think there is no other solution than scaling images manually. Maybe we can talk next week about that.